### DISCUSSION TITLE
목차 수정에 대해서

### DISCUSSION THREAD

**Discussion by snowmerak** [ultimate-go-korean/translation#107] [Ideas]
<img width="247" alt="스크린샷 2021-03-20 오후 9 06 35" src="https://user-images.githubusercontent.com/50787930/111868921-2a387000-89c0-11eb-965c-83aa7333d238.png">

현재 깃북에서 목차는 이런식으로 각 폴더의 readme.md 의 첫번째 타이틀을 기준으로 작성됩니다.  
당장에야 번역 파트를 나누기 위해 이렇게 작성했었지만 일단은 번역이 마무리가 된 시점에 이렇게 나누어 놓기보다 각 부분의 제목을 써두는 게 더 좋다고 생각합니다.  

그에 대한 제 의견으로 기존 readme.md 에 누가 번역하고 누가 리뷰한다는 부분.  
예를 들어 5-20 페이지에는 이런식으로 되어있고
```
Assigned to

- 정현석/@nicewook

Mandatory reviewer

- 이재원/@mumunuu
- 김준희/@juunini
```

그 밑에 간략하게 영어로된 목차가 존재하는데 기여자를 나타낼 수 있는 저 부분만 본문 파일에 옮기고 본문 파일을 readme.md 로 바꾸었으면 합니다.  
그러면서 찾아보니 55-70 처럼 이상하게 작성된 부분도 있으니 수정해갔으면 합니다.

큰 이견이 없다면 이렇게 작성하고 싶은데 다른 의견이나 수정하지 말았으면 하는 이유가 혹시 있을 까요?

**Comment by nicewook** [comment]
이부분 제가 전혀 모릅니당. @@
말씀하신 문제 제기는 이해가 되었습니다. 영민님이 보시고 수정해주실 수 있다면 감히 부탁드리겠습니다. @@
